### DISCUSSION TITLE
Battery life impact of the Android app

### DISCUSSION THREAD

**Discussion by khinsen** [floccusaddon/floccus#1273] [General]
After a few days of testing, I removed the floccus app from my phone because it was causing excessive battery drain (battery lifetime down to 1/3 of my normal rate, back to normal after removing floccus). I don't see anyone else complaining about this, so I wonder on which particularities of my setup this could depend. Android version? System preferences? WebDAV server? Size of bookmark file (unlikely, because mine had only 20 entries)? Other apps installed?

**Comment by marcelklehr** [comment]
The Android app stays active in the background and by default syncs every 15mins. You should be able to stop it manually, though, to save battery.

**Reply by khinsen** [reply]
Thanks, I'll look into that!
